hub/util/tests/test_auto.py

```diff
@@ -13,3 +16,18 @@ def test_most_common_extension(memory_ds: Dataset):
 
     assert compression == "png"
     assert file_compression == "jpeg"
+
+
+def test_ingestion_summary():
+    path = get_dummy_data_path("tests_auto/auto_compression")
```

Comment:
I don't think this test is testing out most of ingestion summary, most of it is still untested.

Reply:
done, I got rid of the preceding and trailing '====' as it was dependent on the terminal window size and the output varied on the ci OS.
now the tests are cleaner and should be good